jsk_arc2017_baxter/euslisp/lib/pick-interface.l

```diff
@@ -130,6 +130,7 @@
     ;; TODO: in-hand verification
     ;; TODO:   target-obj = in-hand object  -> :set_target_cardboard
     ;; TODO:   target-obj != in-hand object -> :return_object
+    (setq target-obj t)
```

Comment:
I think `smach` doesn't die because of it.
what you want is return `t`, so return `t`. 
Do not set `t` in `target-obj`.